.claude/skills/publish-hub-research-run/upload_fix.py

```
@@ -0,0 +1,105 @@
+# Written and run by Claude Fable 5.1 subagents during the 2026-09-04 Hub publication session
+# (rmems/synthetic-factory). Kept as the harness for .claude/skills/publish-hub-research-run.
+#!/usr/bin/env python3
```

Ruff (0.16.3)

[warning] Shebang should be at the beginning of the file

(EXE005)